.circleci/config.yml

```diff
@@ -319,7 +320,7 @@ workflows:
             - nodejs-install
             - team-lumos
             - snyk-bot-slack
-          node_version: "12"
+          node_version: "14"
```

Comment:
can we define this only once?

Reply:
Yes.. I've added `windows_node_version` variable 👍